samples/dnn/trackers/dasiamrpn.py

```diff
@@ -1,39 +1,40 @@
 """
 DaSiamRPN tracker.
-Original paper: https://arxiv.org/abs/1808.06048
+Link to original paper: https://arxiv.org/abs/1808.06048
 Link to original repo: https://github.com/foolwood/DaSiamRPN
 Links to onnx models:
 network:     https://www.dropbox.com/s/rr1lk9355vzolqv/dasiamrpn_model.onnx?dl=0
 kernel_r1:   https://www.dropbox.com/s/999cqx5zrfi7w4p/dasiamrpn_kernel_r1.onnx?dl=0
 kernel_cls1: https://www.dropbox.com/s/qvmtszx5h339a0w/dasiamrpn_kernel_cls1.onnx?dl=0
 """
 
-import numpy as np
-import cv2 as cv
 import argparse
+import cv2 as cv
+import numpy as np
 import sys
 
+
 class DaSiamRPNTracker:
     # Initialization of used values, initial bounding box, used network
     def __init__(self, net="dasiamrpn_model.onnx", kernel_r1="dasiamrpn_kernel_r1.onnx", kernel_cls1="dasiamrpn_kernel_cls1.onnx"):
         self.windowing = "cosine"
         self.exemplar_size = 127
         self.instance_size = 271
-        self.total_stride = 8
-        self.score_size = (self.instance_size - self.exemplar_size) // self.total_stride + 1
+        self.anchor_stride = 8
+        self.anchor_ratios = [0.33, 0.5, 1, 2, 3]
+        self.anchor_scales = [8]
+        self.anchor_num = len(self.anchor_ratios) * len(self.anchor_scales)
         self.context_amount = 0.5
-        self.ratios = [0.33, 0.5, 1, 2, 3]
-        self.scales = [8, ]
-        self.anchor_num = len(self.ratios) * len(self.scales)
+        self.score_size = (self.instance_size - self.exemplar_size) // self.anchor_stride + 1
         self.penalty_k = 0.055
         self.window_influence = 0.42
         self.lr = 0.295
-        self.score = []
         if self.windowing == "cosine":
             self.window = np.outer(np.hanning(self.score_size), np.hanning(self.score_size))
         elif self.windowing == "uniform":
             self.window = np.ones((self.score_size, self.score_size))
         self.window = np.tile(self.window.flatten(), self.anchor_num)
+        self.score = []
```

Comment:
I think this change unnecessary

Reply:
Is it normal to have two similar classes to have different names for the same attributes?

Reply:
score will be removed in later commits